react/components/ProductDescription/Wrapper.js

```diff
@@ -1,17 +1,12 @@
 import React, { useContext } from 'react'
 import { ProductContext } from 'vtex.product-context'
-import { path, isEmpty } from 'ramda'
+import { path } from 'ramda'
 
 import ProductDescription from './index'
 
 const ProductDescriptionWrapper = props => {
   const valuesFromContext = useContext(ProductContext)
-
-  const isContextEmpty = !valuesFromContext || isEmpty(valuesFromContext)
-
-  const description = isContextEmpty
-    ? props.description
-    : path(['product', 'description'], valuesFromContext)
+  const description = props.description != null ? props.description : path(['product', 'description'], valuesFromContext)
```

Comment:
```suggestion
  const description = props.description || path(['product', 'description'], valuesFromContext)
```

Reply:
Why you are making this check here? I understand it on the `availability` case, but not here